docs/edgeanalytics/kafka-options.md

```diff
@@ -0,0 +1,186 @@
+            ssl.endpoint.identification.algorithm=https
+            ssl.truststore.type=JKS
+            ssl.truststore.password=trustpassword
+            ssl.truststore.location={applicationDir}/etc/truststore.jks
+            ```
+
+        1. Copy the JKS files and `kafka.properties` to `etc/` in the SPL application workspace
+
+For more information, see [Using streamsx.kafka with Red Hat AMQ Streams](https://ibmstreams.github.io/streamsx.kafka/docs/user/UsingRHAmqStreams/) documentation.
+
+***
+
+## Event Streams in IBM Cloud
+
+[IBM Event Streams](https://www.ibm.com/cloud/event-streams) builds on top of open source Apache Kafka to offer enterprise-grade event streaming capabilities.
+
+### Provisioning Event Streams
+
+1. Log in to [IBM Cloud](https://cloud.ibm.com) or create an account if you do not have one.
+1. Visit [Event Streams](https://cloud.ibm.com/catalog/services/event-streams) in the catalog.
+1. Select a region (e.g. Dallas, Frankfurt)
+1. Select a plan (e.g. Lite)
+1. Enter in a service name (e.g. Event Streams for Edge)
+1. Click 'Create'
+
+### Creating credentials and a topic
+
+1. Go to 'Service credentials' in the navigation pane.
+1. Click 'New credential'.
+1. Give the credential a name so you can identify its purpose later. You can accept the default value.
+1. Give the credential the Manager role so that it can access the topics, and create them if necessary.
+1. Click 'Add'. The new credential is listed in the table in Service credentials.
+1. For the newly created credentials, click the 'Copy to clipboard' icon.
+1. Go to the Topics tab.
+1. Go to 'Manage' in the navigation pane.
+1. Click 'Create a topic'
+1. Name your topic.
+1. Keep the defaults set in the rest of the topic creation, click 'Next' and then 'Create topic'.
+
+For any questions or details regarding Event Streams, see the [Event Streams documentation](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-getting_started#getting_started) for more information.
+
+### Connecting to Event Streams
+
+Use the copied credentials to save them to a file or create a [Streams application config](https://www.ibm.com/support/knowledgecenter/SSCRJU_4.3.0/com.ibm.streams.admin.doc/doc/creating-secure-app-configs.html) with the credentials:
+
+* **Python applications**: see [Connecting with the IBM Event Streams cloud service](https://streamsxkafka.readthedocs.io/en/latest/#connection-with-the-ibm-event-streams-cloud-service).
+* **SPL applications**: see [`streamsx.messagehub` samples](https://ibmstreams.github.io/streamsx.messagehub/docs/user/overview/#samples) for connection configuration options.
+
+***
+
+## Vanilla Apache Kafka
+
+Apache Kafka can be deployed on bare metal/VM systems as well as Kubernetes and OpenShift.
+
+Because users should already have access to a Kubernetes-like environment, the following install section will cover Helm charts and Operators to deploy Kafka servers.
+
```

Comment:
Just to make sure I understand, if for some reason they're not using k8s and they have Kafka installed anywhere else, that system must be accessible from the CPD cluster, right?

Reply:
The Kafka system does not need to necessarily be accessable from the CPD cluster. If customers want the data from that Kakfa system to be used in other CPD services, then yes. I'll add a stub about this.